android/build.gradle

```diff
@@ -49,7 +54,7 @@ android {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
From Kotlin 1.8 the jdk suffix is not needed, as both jdk7 and jdk8 have been merged